jsbits/Scenario.js

```diff
@@ -207,9 +221,13 @@ function gm$smartProcessFeatureCollection(fc, defVec, maxGeomId) {
               , gm$resizeX(defVec, transformFunc(cmax))
               , dims
               , center
+              , lat
+              , lon
+              , alt
+              , srid
               ];
     } else {
-      return [points,lines,surfaces,deletes,errors,gm$resizeX(defVec, cmin),gm$resizeX(defVec, cmax),dims, null];
+      return [points,lines,surfaces,deletes,errors,gm$resizeX(defVec, cmin),gm$resizeX(defVec, cmax),dims, null,lat,lon,alt,srid];
```

Comment:
This is good idea:
we parse FeatureCollection and return lat/lon if we can derive it from the data. But be careful with nulls!